### PR TITLE
Multi team fix

### PR DESCRIPTION
**Pull request by aldodfm**
Hello,

With this PR i want to update 2 points:

- Be compatible with fastlane pilot version 1.10.0 (last version), then the automatic choose for TeamID and Team Name now works. You can check [here](https://github.com/fastlane/fastlane/blob/master/pilot/lib/pilot/options.rb)  where Pilot Team ID is "-q" and Pilot Team Name is "-r".

- Fix the problems with TeamNames with Spaces, example: Company Country inc -to-> "Company Country inc".

### CODE REVIEW

File: Tasks/app-store-promote/app-store-promote.ts [hunk 1]
```diff
@@ -39,7 +39,12 @@ async function run() {
         var shouldAutoRelease = tl.getBoolInput('shouldAutoRelease', false);
         var teamId = tl.getInput('teamId', false);
         var teamName = tl.getInput('teamName', false);
-
+        if (teamName)
```
Comment (madhurig) [inline Tasks/app-store-promote/app-store-promote.ts]: Node handles the args with spaces, we should not need the quotes

File: Tasks/app-store-release/app-store-release.ts [hunk 2]
```diff
@@ -55,6 +55,11 @@ async function run() {
         var shouldSkipSubmission: boolean = tl.getBoolInput("shouldSkipSubmission", false);
         var teamId: string = tl.getInput("teamId", false);
         var teamName: string = tl.getInput("teamName", false);
+        if (teamName)
```
Comment (madhurig) [inline Tasks/app-store-release/app-store-release.ts]: Shouldn't need to add the quotes

File: Tasks/app-store-promote/app-store-promote.ts (hunk 1 above)
Comment (aldodfm) [inline Tasks/app-store-promote/app-store-promote.ts]: you are right.

File: Tasks/app-store-release/app-store-release.ts (hunk 2 above)
Comment (aldodfm) [inline Tasks/app-store-release/app-store-release.ts]: that's true.

**Comment by aldodfm** [comment]
i updated it, i removed adding quotes. ;)

**Comment by madhurig** [comment]
@aldodfm: Thanks!

**Pull request merged**